src/js/emscripten-settings.ts

```diff
@@ -5,6 +5,11 @@ import { initializeNativeFS } from "./nativefs";
 import { loadBinaryFile, getBinaryResponse } from "./compat";
 import { API, PreRunFunc, type Module } from "./types";
 
+// @ts-ignore
+import sentinelWasm from "./generated/sentinel.wasm";
+
+declare const sentinelWasm: Uint8Array;
```

Comment:
I don't see any place where this variable is used. Is it for side effect?

Reply:
We `WebAssembly.compile()` it. I moved it to `sentinel.ts` which should be a bit easier to read hopefully.